<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Date</class>
 <widget class="QMainWindow" name="Date">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>网卡名称：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>过滤器</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="10" colspan="3">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ip:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_ip">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>arp:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_arp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>rarp:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_rarp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>tcp:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_tcp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>udp:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_udp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" rowspan="4" colspan="3">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="11" column="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>icmp:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QLabel" name="label_icmp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>other:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="label_o">
      <property name="text">
       <string>other:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
